### ISSUE TITLE
Medaka_consensus python command not found error message

### ISSUE THREAD

**Issue by LEGMCGILL** [nanoporetech/medaka#481]
**Describe the bug**
I am trying to do self polishing of my nanopore data, however, I am encountering python: command not found error message, although python is installed, I have python3 and python3.8 apparently. I have tried a couple of options including moving the medaka_consensus to my path, I also tried moving python to my path.
Find the following picture with the error message that I am getting:
Some things I have tried with no changes is updating from medaka 1.11.2 to medaka 1.11.3
**Logging**
No relevant logging messages.

**Environment (if you do not have a GPU, write No GPU):**
 - Installation method [from pypi: pip install medaka]
 - OS: Ubuntu 20.04
 - medaka version (1.11.2)

There is no GPU in the instance I am currently working on, I dont have a Nvidia driver, nor CUDA or cuDNN installed.

**Additional context**
Find the picture of my screen with the commands as well as the error messages I am receiving. 

Best,

Laura.
![image](https://github.com/nanoporetech/medaka/assets/147701622/15d6b2d0-1058-4f57-a544-9291623e6d88)

**Comment by cjw85** [comment]
On Ubuntu20.04 the default setup is that the Python interpreter is available only through explicit use of the `python3` command, not `python`. There is a system package `python-is-python3` that can be installed in order to make `python` refer to `python3`. This can be installed with:

```
sudo apt install python-is-python3
```

We also recommend using a virtual environment to install medaka. For this on Ubuntu 20.04 you will first need to install the system package:

```
sudo apt install python3-virtualenv
```

as the Ubuntu maintainers choose not to include the Python virtualenv component as part of their Python distribution.

To install medaka then run:

```
python3 -m venv medaka --prompt "(medaka) "
. medaka/bin/activate
pip install --upgrade pip
pip install medaka
```

(similar to those commands in the README). Whenever you wish to run medaka you will always need to activate the virtual environment with:

```
. medaka/bin/activate
```

We recommend the use of virtual environments to maintain isolation between different software installations.

**Comment by LEGMCGILL** [comment]
Thank you so much, 

This instructions where really helpful to sort the initial error message. I also installed all of the libraries and packages needed to run along medaka. At the moment of running the analysis it starts correctly the process, and it keeps on displaying the process being done but at the end it finally displays a message saying that medaka consensus failed to run. 

See attached picture. 

![image](https://github.com/nanoporetech/medaka/assets/147701622/1bfe617b-e695-4fc7-9123-fe4771bfbb27)
I also noticed that some files where generated even with this error message but I am assuming they are still incomplete? 
Are they the usual output one is supposed to observe or are they all complete regardless of the message I received. 

Thanks for your valuable feedback. 


![image](https://github.com/nanoporetech/medaka/assets/147701622/e9071b97-8300-486c-a546-02400fe9fb53)

Laura.
